experimental/Schemes/IdealSheaves.jl

```diff
@@ -793,6 +793,17 @@ end
   return IdealSheaf(X, ID, check=false)
 end
 
+function small_generating_set(II::IdealSheaf)
+    X = scheme(II)
+  # If there is a simplified covering, do the calculations there.
+  covering = (has_attribute(X, :simplified_covering) ? simplified_covering(X) : default_covering(X))
+  ID = IdDict{AbsSpec, Ideal}()
+  for U in patches(covering)
+    ID[U] = ideal(base_ring(II(U)),Oscar.small_generating_set(saturated_ideal(II(U))))
+  end
+  return(IdealSheaf(X, ID, check = false))
+end
```

Comment:
Please fix the indentation. Also the `Oscar.` should not be necessary.

Reply:
done.